analyzer/src/funTest/kotlin/managers/PubFunTest.kt

```diff
@@ -113,4 +141,34 @@ class PubFunTest : WordSpec() {
         val config = AnalyzerConfiguration(allowDynamicVersions = true)
         return Pub("Pub", USER_DIR, config, DEFAULT_REPOSITORY_CONFIGURATION)
     }
+
+    /**
+     * Replace aapt2 URL and hash value with dummy values, as these are platform dependant.
+     */
+    private fun OrtResult.patchAapt2Result(): OrtResult {
+        val packages = analyzer?.result?.packages?.toMutableSet()
+        val aapt2Package =
+            packages?.find { it.pkg.id == Identifier("Maven:com.android.tools.build:aapt2:4.1.0-6503028") }
```

Comment:
Should this match any version of the library in case someone has a different SDK version installed or is this version defined by the project?

Reply:
Removed the version from the comparison.